x-pack/plugins/security_solution/server/lib/detection_engine/signals/utils.ts

```diff
@@ -120,8 +120,10 @@ export const hasTimestampFields = async (
     // if there is a timestamp override and the unmapped array for the timestamp override key is not empty,
     // partial failure
     const errorString = `The following indices are missing the ${
-      timestampField === '@timestamp' ? 'timestamp field "@timestamp"' : 'timestamp override field'
-    } "${timestampField}": ${JSON.stringify(
+      timestampField === '@timestamp'
+        ? 'timestamp field "@timestamp"'
+        : `timestamp override field "${timestampField}"`
+    }: ${JSON.stringify(
```

Comment:
Would it be a big undertaking to test this message somewhere? Not critical, but it'd be nice to have.

Reply:
Yeah for sure I think that's a good little test to add. I'll update this.